src/main/java/com/gluonhq/richtext/RichTextArea.java

```diff
@@ -78,6 +80,31 @@ public final int getTextLength() {
         return textLengthProperty.get();
     }
 
+    // contentAreaWidthProperty
+    /**
+     * Defines a width constraint for the content area of the rich text control,
+     * in user space coordinates, where text can be added.
+     * The width is measured in pixels (and not glyph or character count).
+     * If the value is {@code == 0}, the content area extends to the whole viewport of control
+     * and will change whenever the viewport gets resized.
```

Comment:
Shouldn't this happen for any value less than zero? If yes, then the javadoc needs to be updated.

Reply:
So far <0 values aren't expected. But just in case, I'll modify it.